test/corpus/rbs_annotations.txt

```diff
@@ -0,0 +1,231 @@
+=======================
```

Comment:
Could you add a test showing the behavior if the `#:` is found inside a string:

```rb
puts "#: Hello, World"
```

Reply:
Done!